clients/csharp-dotnet-core/voice-assistant-test/README.md

```diff
@@ -165,6 +165,9 @@ Here is the full list:
 >>##### WavFile
 >>` string | optional | null | "test1.WAV"`. Audio from this WAV file is streaming to Direct Line Speech as the input in the turn, by calling the [ListenOnceAsync](https://docs.microsoft.com/en-us/dotnet/api/microsoft.cognitiveservices.speech.dialog.dialogserviceconnector.listenonceasync?view=azure-dotnet) method (or [StartKeywordRecognitionAsync](https://docs.microsoft.com/en-us/dotnet/api/microsoft.cognitiveservices.speech.dialog.dialogserviceconnector.startkeywordrecognitionasync?view=azure-dotnet) method if [Keyword](#keyword) is true). This represents a user speaking to a microphone. It's good practice to have at least one second of silence (non speech) at the end of the WAV file to make sure the speech service properly detects end-of-speech, as it would with a live audio stream from a microphone. When this field is present, you can specify the expected recognition result in the [Utterance](#utterance) field.
 >>
+>>##### WavFileDuration
+>>` int | optional | null | 3000`. Duration of input WavFile in milliseconds.

```

Comment:
We never got to finish the discussion about the two option to specify the speech duration: either as a seperate field, or as a number added (with comma) after teh WavFile name.
Since this new duration value goes together with a wav file, I'm include to addome the "comma + duration value" at the end of the WavFile string. The same way we do it for ExpectedResponseLatency. A bit harder to parse, but easier to document and makes it clear that those two come togehter. What do you think?